src/server/api/parameters_parser.cpp

```diff
@@ -55,6 +55,11 @@ boost::optional<ParameterT> parseParameters(std::string::iterator &iter,
         // iterator to the failing position. Extract the position from the exception ourselves.
         iter = failure.first;
     }
+    catch (const boost::numeric::bad_numeric_cast &e)
+    {
+        // this can happen if we get bad numeric values in the request, just handle
+        // as normal parser error
+    }
```

Comment:
Invalid coordinates are caught at a higher level. This only fixes the case where the actual numeric value does not fit in the data type. I added another test there to demonstrate this.